eisbuk-admin/esbuild/src/lib/config.ts

```diff
@@ -0,0 +1,19 @@
+import { BuildOptions } from "esbuild";
+import path from "path";
+import svgr from "esbuild-plugin-svgr";
+
+/**
+ * A base esbuild config file shared between both `build` and `serve`
+ * scripts.
+ */
+const config: BuildOptions = {
+  entryPoints: [path.join(process.cwd(), "src", "index.tsx")],
```

Comment:
Using a path relative to the current file would work even when the build is invoked from an unusual directory.

Reply:
Nevermind:

* the current directory is changed by yarn when invoking `yarn build` to be `eisbuk-admin`
* the `config.ts` file is executed by reading it into a buffer and running `eval` on it, so we don't really have access to `__dirname` or `__filename` there